rosbag2_transport/src/rosbag2_transport/recorder.cpp

```diff
@@ -83,4 +75,65 @@ Recorder::create_subscription(
   return subscription;
 }
 
+std::future<void> Recorder::launch_topics_discovery(
+  std::chrono::milliseconds topic_polling_interval,
```

Comment:
i think it makes sense to actually use ROS time (rclcpp time). so we can react on system clocks vs steady clock vs sim time.

Reply:
I think I do not understand what the possible advantage would be. The polling interval controls how often rosbag looks for new topics. I would expect this to happen in regularly in "rosbag"-time independent of what the ros time is "doing".